src/data/pnat.lean

```diff
@@ -5,13 +5,17 @@ Author: Mario Carneiro
 -/
```

Comment:
I think it would be very appropriate to add your name to the list of authors :smile: